use-timescale/data-tiering/disabling-data-tiering.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Creating a tiering policy
+excerpt: How to create a tiering policy
```

Comment:
```suggestion
title: Disabling tiering for a hypertable
excerpt: How to disable tiering for a hypertable
```

Reply:
Disable tiering for a hypertable


No ing words please :-).

Reply:
fixed.